File: example/reducer.py
from org.apache.hadoop.io import Text

_rkey = Text()
_rvalue = Text()

def reduce(key,values,context):
    count = 0
    for value in values:
        count+=1
    _key.set(key)
    _value.set(str(count))
    context.write(_key,_value)

File: phadoop-js/src/test/resources/reducer.js
_rkey = new org.apache.hadoop.io.Text();
_rvalue = new org.apache.hadoop.io.Text();

function reduce(key, values, context){
    var iterator=values.iterator();
    while (iterator.hasNext()) {
        _key.set(key);
        _value.set(iterator.next());
        context.write(_key,_value);
    }
}

